fetch/api/basic/stream-response.any.js

```diff
@@ -1,7 +1,8 @@
 // META: global=window,worker
 // META: script=../resources/utils.js
 
-function streamBody(reader, test, count) {
+function streamBody(reader, test) {
+  let count = 0;
```

Comment:
I think you need to keep this as a third argument with a default value, given the recursive invocation. Or am I missing something?

Reply:
Oh right... not properly awake yet it seems.